ProtocolConformance.md

```diff
@@ -1,146 +1,294 @@
 # 1 Protocol Conformance
+When a named concrete type[^1] `T` is conformed to a protocol `P`, a set of implementations--*witnesses*--is 
+determined, one for each requirement of `P`.   That set of witnesses is referred to as the protocol conformance
+of `T: P`.  This document specifies how Swift determines a conformance.
+[^1]: What about tuples and [SE-0283](https://github.com/apple/swift-evolution/blob/master/proposals/0283-tuples-are-equatable-comparable-hashable.md)?
```

Comment:
I like that approach.

For your consideration, I'd like to suggest:
- a `typealias` (declared or inferred) is the witness for an `associatedtype`
- for requirements, we should distinguish between `func` and `var` requirements
- we could use the term instance rather than non-`static`

Reply:
* typealiases really *are* aliases for types, rather than types themselves.  I think the witness is the actual type being aliased.  The typealias is just the thing that connects the witness to the conforming type.
* Ah yes, I left out properties.  And subscripts.  The witnesses are are actually getter functions or getter/setter pairs. I'lll write some text for that once we get this a little more settled.  We should drive this toward a state where it can be landed in master even if incomplete.
* “instance” works for me.
* I notice I also left out generic method /subscript requirements, whose instances are actually generics.